src/rgw/rgw_asio_frontend.cc

```diff
@@ -138,6 +138,7 @@ void handle_connection(boost::asio::io_context& context,
                        boost::system::error_code& ec,
                        spawn::yield_context yield)
 {
+  init_jager();
```

Comment:
inside this func there is atomic boolean which checks if a tracer has been initialized or not, I cannot put it in common_init because those are called only once when ceph clusters starts and if while starting ceph cluster the backend server of tracer is unreachable then it wont show any spans, untill unless the ceph cluster is restarted again to establish the connection

Reply:
>i suggested using a string literal or __PRETTY_FUNCTION__ directly. i don't like how this adds 2 lines of boilerplate to >every trace point - i'd really like to get it down to a single line